lib/toucan.js

```diff
@@ -34,7 +34,7 @@ async function prepareToucanEnv(admin, broker) {
 
 	let data = {};
 	for (const project of genesis) {
-		const projectId = await deployProject(contracts.projects, project, admin.address);
+		const projectId = await deployProject(contracts.projects, project, admin.address, admin.address);
```

Comment:
since this is only used in testing you can set pretty much anything here. the most logical thing would be to use the null address `0x0000000000000000000000000000000000000000`. just to give you context what the beneficiary is about i'll paste Michalis' commit message :) 
> Enable setting a beneficiary in a carbon project
to allow fees of redeemed TCO2s from carbon pools
to flow back to project beneficiaries (royalties).